cms/djangoapps/contentstore/tasks.py

```diff
@@ -125,3 +126,17 @@ def push_course_update_task(course_key_string, course_subscription_id, course_di
     # TODO Use edx-notifications library instead (MA-638).
     from .push_notification import send_push_course_update
     send_push_course_update(course_key_string, course_subscription_id, course_display_name)
+
+
+@task()
+def async_export_to_git(course_module, user=None):
+    """
+    Exports a course to Git.
+    """
+    try:
+        export_to_git(course_module.id, course_module.giturl, user=user)
```

Comment:
my response to this comment was hidden when i changed the base branch of this PR. here's my original response: https://github.com/mitodl/edx-platform/pull/13#discussion_r125070010